RLTest/redis_std.py

```diff
@@ -25,19 +25,8 @@ def __init__(self, redisBinaryPath, port=6379, modulePath=None, moduleArgs=None,
         self.redisBinaryPath = os.path.expanduser(redisBinaryPath) if redisBinaryPath.startswith(
             '~/') else redisBinaryPath
 
-        if modulePath:
-            if not isinstance(modulePath, list):
-                modulePath = [modulePath]
-            self.modulePath = list(map(lambda p: os.path.abspath(p), modulePath))
-        else:
-            self.modulePath = None
-        if isinstance(moduleArgs, list):
-            self.moduleArgs = moduleArgs
-        else:
-            self.moduleArgs = [moduleArgs]
-            if isinstance(modulePath, list) and len(modulePath) > 1:
-                self.moduleArgs.extend([''] * (len(modulePath) - 1))
-
+        self.modulePath = fix_modules(modulePath)
+        self.moduleArgs = fix_modulesArgs(self.modulePath, moduleArgs)
```

Comment:
Why it need to be fixed again?

Reply:
We get `modulePath` and `moduleArgs` from the constructor, thus we need to get them to the standard representation.